gateway/event_handler_jsvm.go

```diff
@@ -17,15 +19,28 @@ type JSVMContextGlobal struct {
 
 // JSVMEventHandler is a scriptable event handler
 type JSVMEventHandler struct {
-	methodName string
-	Spec       *APISpec
-	SpecJSON   string
-	Gw         *Gateway `json:"-"`
+	conf     apidef.JSVMEventHandlerConf
+	Spec     *APISpec
+	SpecJSON string
+	Gw       *Gateway `json:"-"`
 }
 
-// New enables the intitialisation of event handler instances when they are created on ApiSpec creation
-func (l *JSVMEventHandler) Init(handlerConf interface{}) error {
-	l.methodName = handlerConf.(map[string]interface{})["name"].(string)
+// Init initializes the JSVMEventHandler by setting the method name and global values required for the JavaScript VM.
+func (l *JSVMEventHandler) Init(handlerConf any) error {
+	var err error
+	if err = l.conf.Scan(handlerConf); err != nil {
+		log.WithFields(logrus.Fields{
+			"prefix": "jsvm_event_handler",
+		}).Error("Problem getting configuration, skipping. ", err)
```

Comment:
```suggestion
		l.Gw.Logger().WithError(err).Error("Problem getting configuration, skipping.")
```

- drops usage of global `log`
- Logger implementation drops WithPrefix noise
- correct use of WithError(err)

repeats

Reply:
Following up with TT-14215